app/models/event_procedure.rb

```diff
@@ -31,4 +31,15 @@ class EventProcedure < ApplicationRecord
   validates :room_type, presence: true, if: -> { health_insurance? }
   validates :urgency, inclusion: [true, false], if: -> { health_insurance? }
   validates :payment, presence: true
+
+  validate :unique_user
+
+  private
+
+  def unique_user
```

Comment:
find a better name to this ,ethod
```suggestion
  def unique_user
```